### DISCUSSION TITLE
Regarding project

### DISCUSSION THREAD

**Discussion by dhfdhdfh** [HARAMIS14/AIML-Basic-College-Enquiry-Chatbot#6] [General]
Hi..am Ashif.. can I get detailed review or screenshot on this project? 
My mail is- www.ashifpanthalookaran@gmail.com

Please reply

**Comment by HARAMIS14** [comment]
Thank you for asking for details Ashif!! 
We will mail it to you shortly. 
Regards,
Team HARAMIS

**Comment by dhfdhdfh** [comment]
Thank you for your response , you can send it here 🙂
